hack/table-gen/table-gen.go

```diff
@@ -0,0 +1,242 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"sort"
+	"strings"
+
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	REPattern                   = `(?s)<!--\s*TABLE-START\s* -->.*<!--\s*TABLE-END\s*-->`
+	SkipIdentifier              = `SKIP-ELEMENT`
+	RESkipPattern               = `<!--\s*` + SkipIdentifier + `\s*([^\s]+)\s*-->`
+	SkipWithAncestorsIdentifier = `SKIP-WITH-ANCESTORS`
+	RESkipWithAncestorsPattern  = `<!--\s*` + SkipWithAncestorsIdentifier + `\s*([^\s-]+)\s*-->`
+)
+
+var (
+	CRDFilename string
+	MDFilename  string
+	APIVersion  string
+	CRDKind     string
+)
+
+func main() {
+	flag.StringVar(&CRDFilename, "crd-filename", "", "Full or relative path to the .yaml file containing crd")
+	flag.StringVar(&MDFilename, "md-filename", "", "Full or relative path to the .md file containing the file where we should insert table rows")
+	flag.StringVar(&APIVersion, "api-version", "v1alpha1", "API version your operattor uses")
```

Comment:
```suggestion
	flag.StringVar(&APIVersion, "api-version", "v1alpha1", "API version your operator uses")
```

Reply:
The version is not used in your scripts. Please specify them in the script as well and use it in the generation marker. We soon will have multiple versions